doc/api/cli.md

```diff
@@ -155,6 +158,22 @@ V8 inspector integration allows tools such as Chrome DevTools and IDEs to debug
 and profile Node.js instances. The tools attach to Node.js instances via a
 tcp port and communicate using the [Chrome DevTools Protocol][].
 
+<a id="inspector_security"></a>
+#### Warning: binding inspector to a public IP:port combination is insecure
+
+Binding the inspector to a public IP (including `0.0.0.0`) with an open port is
+insecure, as it allows third-party hosts to connect to the inspector and perform
```

Comment:
Wouldn't external hosts be better than third-party here? We don't even have a second party here.

Reply:
Will fix, thanks!